db/migrate/20240120015121_create_expenses.rb

```diff
@@ -0,0 +1,15 @@
+class CreateExpenses < ActiveRecord::Migration[7.1]
+  def change
+    create_table :expenses do |t|
+      t.date :date
+      t.string :day
```

Comment:
Is this really required, as month is not included. Can it be extracted from date?